scripts/mirror_lkl_layer.py

```diff
@@ -0,0 +1,132 @@
+"""
+   Copyright 2020 Esri
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.​
+
+    This sample mirrors LKL data from a Location Tracking Service or Location Tracking View into a standard feature service.
+    That allows the user to support dynamic joins of data.
+"""
+import argparse
+import logging
+import logging.handlers
+import traceback
+import sys
+from arcgis.gis import GIS
+from arcgis.features import FeatureLayer
+
+
+def initialize_logging(log_file=None):
+    """
+    Setup logging
+    :param log_file: (string) The file to log to
+    :return: (Logger) a logging instance
+    """
+    # initialize logging
+    formatter = logging.Formatter(
+        "[%(asctime)s] [%(filename)30s:%(lineno)4s - %(funcName)30s()][%(threadName)5s] [%(name)10.10s] [%(levelname)8s] %(message)s")
+    # Grab the root logger
+    logger = logging.getLogger()
+    # Set the root logger logging level (DEBUG, INFO, WARNING, ERROR, CRITICAL)
+    logger.setLevel(logging.DEBUG)
+    # Create a handler to print to the console
+    sh = logging.StreamHandler(sys.stdout)
+    sh.setFormatter(formatter)
+    sh.setLevel(logging.INFO)
+    # Create a handler to log to the specified file
+    if log_file:
+        rh = logging.handlers.RotatingFileHandler(log_file, mode='a', maxBytes=10485760)
+        rh.setFormatter(formatter)
+        rh.setLevel(logging.DEBUG)
+        logger.addHandler(rh)
+    # Add the handlers to the root logger
+    logger.addHandler(sh)
+    return logger
+
+
+def return_field_name(layer, name_to_check):
```

Comment:
I don't think this is needed since the Global ID field is always `globalid`

Reply:
For a standard hosted feature service, I see `GlobalID` in AGOL and `globalid` in Enterprise.

Reply:
Oh yeah that's true